db/migrate/20160324114924_add_index_to_create_subjects.rb

```diff
@@ -0,0 +1,6 @@
+class AddIndexToCreateSubjects < ActiveRecord::Migration
+  def change
+    add_column :create_subjects, :name, :string
+    add_index :create_subjects, :name, unique: true
```

Comment:
Does this work? I doubt it does because you don't have a table. Also the column name is wrong.